app/src/main/java/com/example/superimagesviewer/repository/MosaicRepository.kt

```diff
@@ -18,23 +19,52 @@ import org.json.JSONObject
 
 class MosaicRepository(application: Application) {
 
-    val TAG = MosaicRepository::class.simpleName
+    private val TAG = MosaicRepository::class.simpleName
 
-    private val drawables = mutableListOf<Drawable>()
+    private val drawables = mutableListOf<InstagramImage>()
```

Comment:
Please rename to something like "instagramImageList"

Reply:
done